x-pack/plugins/actions/server/usage/actions_usage_collector.ts

```diff
@@ -23,6 +23,8 @@ export function createActionsUsageCollector(
       return true;
     },
     schema: {
+      has_errors: { type: 'boolean' },
+      error_messages: { type: 'array', items: { type: 'keyword' } },
```

Comment:
I would add an `ignore_above` to this mapping: https://www.elastic.co/guide/en/elasticsearch/reference/current/ignore-above.html. Maybe limit to 1024. If the error strings get too long the indexing will fail if there is no `ignore_above` specified

Reply:
it doesn't allow me to add that, IDK why. 
TS complains with:
```
Type '{ type: "keyword"; ignore_above: number; }' is not assignable to type '{ type: AllowedSchemaStringTypes; _meta?: { description: string; } | undefined; }'.   Object literal may only specify known properties, and 'ignore_above' does not exist in type '{ type: AllowedSchemaStringTypes; _meta?: { description: string; } | undefined; }'.
```

Reply:
@pgayvallet Do we not allow `ignore_above` in the telemetry mappings?